source/includes/api/_dataset.md

```diff
@@ -810,9 +1051,7 @@ For more details on the characteristics of each of the basic data types, refer t
 
 ## Uploading a Dataset (Binary)
 
-To upload a dataset, you will need to be authenticated.
-
-You can upload your raw data directly to S3 making use of the "upload" endpoint. This endpoint accepts a file in the property "dataset" and returns a valid connectorUrl. With this connectorUrl you can create or update a "document" dataset, or a raster dataset in the Rasdaman adapter.
+> Upload raw data for using in a dataset
```

Comment:
I'm confused as to how this relates to the "Creating a dataset" section above. Reading the example code makes it clearer, but could use more description.